src/util-radix-tree.c

```diff
@@ -963,14 +961,15 @@ SCRadixNode *SCRadixAddKeyIPV6Netblock(uint8_t *key_stream, SCRadixTree *tree,
  *
  * \retval node Pointer to the newly created node
  */
-SCRadixNode *SCRadixAddKeyIPV4String(const char *str, SCRadixTree *tree, void *user)
+bool SCRadixAddKeyIPV4String(const char *str, SCRadixTree *tree, void *user, bool *is_duplicate)
```

Comment:
alternative approach would be to use the `sc_errno` logic to set a `SC_EEXIST`-style errno